__tests__/client.basic.test.ts

```diff
@@ -307,6 +340,10 @@ describe("MultiServerMCPClient", () => {
           transport: "sse",
           url: "http://localhost:8000/sse",
         },
+        server3: {
+          transport: "streamable",
```

Comment:
(nit) why "streamable" and not "streamable-http" or "streamable_http"?

Reply:
I didn't decide the key - honestly I think `http` is better since this option should be used for all HTTP endpoints (including SSE). Will change to that.

Reply:
Changed - will likely need to add strings to this union as other popular client configs come out and decide they want to use some other key, however.